topic/tests/index.md

```diff
@@ -508,4 +508,22 @@ Sometimes a directive may call an internal function that performs specific actio
 With the above example, we could combine the data sets with this example to provide even shorter code, but it may be undesirable to do so.  For instance, with this code structure one can assert additional effects happened or mix in several `beforeEach()` calls.  Remember that the goal is maintainable, clean and self-describing code.
 
 
+Functional Testing
+------------------
+
+Functional testing is the art of testing how your website or api works from the eye of a user, or at least with everything put together. It is very similar to unit testing in the way of how tests are laid out, but there are some things to watch out for.
+
+First, we want to load the page or refresh each time a test is started. This is to make sure we carry over any data from one test to another. For instance, if data was set to be used in the `session` we want to make sure it gets cleared out before loading the next test. This can usually be done in the `afterEach()` call using `window.sessionStorage.clear();`.
```

Comment:
Is the second sentence supposed to be: "this is to make sure we DON'T carry over any data from one test to another"?